### PR TITLE
GW-47: merge sponsorship page to the current website

### PR DESCRIPTION
**Pull request by chrishadrian**
- Added partners page with its responsive views
- Restructured and refactored some files

### CODE REVIEW

File: src/pages/partners/components/SponsorList.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                    </Grow>
+                    {/* Conditionally applies the timeout prop to change the entry speed. */}
+                    <Grow
+                        in={checked}
+                        style={{ transformOrigin: '0 0 0', width: "100%" }}
+                        {...(checked ? { timeout: 3000 } : {})}
+                    >
+                        {growSponsorList(PARTNERS.slice(5))}
+                    </Grow>
+                    <Button variant="text" onClick={handleChange} sx={buttonSx}>SHOW LESS</Button>
```
Comment (Julian-UBC) [inline src/pages/partners/components/SponsorList.js:54]: Try using our Button component instead of MUI's

**Review by Julian-UBC — CHANGES_REQUESTED** [review]

File: src/pages/partners/components/SponsorIndex.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { graphql, useStaticQuery } from "gatsby"
+import { GatsbyImage } from "gatsby-plugin-image"
```
Comment (Julian-UBC) [inline src/pages/partners/components/SponsorIndex.js:2]: We'll need to remove these Gatsby dependencies.

File: src/pages/partners/components/SponsorList.js (hunk 1 above)
Comment (chrishadrian) [reply src/pages/partners/components/SponsorList.js]: Done

File: public/index.html [hunk 3]
```diff
@@ -2,19 +2,19 @@
...
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/gisau_icon.svg" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon-32x32.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Web site created by ❤️"
```
Comment (Julian-UBC) [inline public/index.html:10]: Thank you for making these changes!!

File: src/pages/partners/Partners.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { MainContainer } from "../../shared/layout/MainContainer";
+import { Box } from "@mui/material";
+import { Footer } from "../../shared/components/footer/Footer";
+import { SponsorDivider, SponsorRegistration, SponsorTitle, SponsorList } from "./components/index";
+import { ScrollButtonContainer } from "shared/components/ScrollButton";
+// import { LandingImage } from "pages/home/components/LandingImage";
```
Comment (Julian-UBC) [inline src/pages/partners/Partners.js:7]:
```suggestion

```

File: src/pages/partners/Partners.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+export const Partners = () => {
+    return (
+        <Box className="bg-[#FFFDF5]">
+            {/* <LandingImage /> */}
```
Comment (Julian-UBC) [inline src/pages/partners/Partners.js:12]:
```suggestion

```

File: src/pages/partners/components/SponsorDivider.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import { Divider } from "@mui/material";
+import React from "react";
+
+const SponsorDivider = () => {
```
Comment (Julian-UBC) [inline src/pages/partners/components/SponsorDivider.js:4]: Why do we need this? Why don't we just put the Divider straight in the Sponsor?

**Review by Julian-UBC — APPROVED** [review]

File: src/pages/partners/components/SponsorIndex.js (hunk 2 above)
Comment (chrishadrian) [reply src/pages/partners/components/SponsorIndex.js]: Resolved

File: src/pages/partners/components/SponsorDivider.js (hunk 6 above)
Comment (chrishadrian) [reply src/pages/partners/components/SponsorDivider.js]: I'm not sure myself. Gotta ask myself from 6 months ago.

Comment (chrishadrian) [reply src/pages/partners/components/SponsorDivider.js]: Resolved

**Review by gisauweb — APPROVED** [review]
Good to go!

**Pull request merged**